servers/c/src/cbl/CBLManager.cpp

```diff
@@ -28,6 +28,28 @@ using namespace ts::support::error;
 #define ASSET_BLOBS_DIR "blobs"
 
 namespace ts::cbl {
+    static FLSliceResult xor_cipher(FLSlice input) {
```

Comment:
Ugg... what is this doing?

Reply:
Just something that turns the property into anything but plaintext